tdrs-backend/tdpservice/data_files/admin/admin.py

```diff
@@ -37,6 +37,21 @@ class Media:
 
     actions = ['reparse']
 
+    def get_queryset(self, request):
+        """Return the queryset."""
+        qs = super().get_queryset(request)
+        # return data files based on user's section
+        FRA_SECTION_LIST = [
+                DataFile.Section.FRA_WORK_OUTCOME_TANF_EXITERS,
+                DataFile.Section.FRA_SECONDRY_SCHOOL_ATTAINMENT,
+                DataFile.Section.FRA_SUPPLEMENT_WORK_OUTCOMES
+                ]
+        if not request.user.has_fra_access:
+            filtered_for_fra = qs.exclude(section__in=FRA_SECTION_LIST)
+            return filtered_for_fra
+        else:
+            return qs.filter(section__in=FRA_SECTION_LIST)
```

Comment:
not sure if this is intended, but setting `fra_access=True` means that the data files page filters down to only FRA files. you have to set `fra_access=False` to see other file types. maybe we should have a "both" option for admins and other users so that they don't have to switch the feature flag to find files in the admin